packages/onboarding/src/navigator/navigator-item-group/style.scss

```diff
@@ -0,0 +1,30 @@
+@import "@automattic/typography/styles/fonts";
+
+.navigator-item-group {
+	flex-shrink: 0;
+
+	.navigator-button__wrapper {
+		display: flex;
+
+		.navigator-button {
+			flex: 1;
+			// Expand sides out of wrapper in main screen
+			margin: 0 -11px;
+		}
+	}
+
+	&:nth-child(1 of &):nth-last-child(1 of &) {
```

Comment:
The `:only-child` doesn't select the class name, so replace it with the current way